hypervisor/include/lib/spinlock.h

```diff
@@ -20,12 +20,12 @@ typedef struct _spinlock {
 } spinlock_t;
 
 /* Function prototypes */
-static inline void spinlock_init(spinlock_t *lock)
+static inline void spinlock_init(const spinlock_t *lock)
```

Comment:
We should not add `const` here since the object pointed by `lock` is being modified.

Reply:
OK. I will change it back.